src/main/java/net/openhft/chronicle/hash/impl/InternalMapFileAnalyzer.java

```diff
@@ -296,31 +296,35 @@ static long output64(ByteBuffer buffer, final String tag) {
     }
 
     static void output(final long addr, final String tag, final long out) {
-        System.out.format("%08X - %08X  %-30s: 0x%016X (%d)%n", addr, addr + Long.BYTES - 1, tag, out, out);
+        outputFormat("%08X - %08X  %-30s: 0x%016X (%d)%n", addr, addr + Long.BYTES - 1, tag, out, out);
     }
 
     static void output24u(final long addr, final String tag, final long out) {
-        System.out.format("%08X - %08X  %-30s: 0x%06X           (%d)%n", addr, addr + 3 - 1, tag, out, out);
+        outputFormat("%08X - %08X  %-30s: 0x%06X           (%d)%n", addr, addr + 3 - 1, tag, out, out);
     }
 
     static void output32u(final long addr, final String tag, final long out) {
-        System.out.format("%08X - %08X  %-30s: 0x%08X         (%d)%n", addr, addr + 4 - 1, tag, out, out);
+        outputFormat("%08X - %08X  %-30s: 0x%08X         (%d)%n", addr, addr + 4 - 1, tag, out, out);
     }
 
     static void output40u(final long addr, final String tag, final long out) {
-        System.out.format("%08X - %08X  %-30s: 0x%010X       (%d)%n", addr, addr + 5 - 1, tag, out, out);
+        outputFormat("%08X - %08X  %-30s: 0x%010X       (%d)%n", addr, addr + 5 - 1, tag, out, out);
     }
 
     static void output(final long addr, final String tag, final int out) {
-        System.out.format("%08X - %08X  %-30s: 0x%08X         (%d)%n", addr, addr + Integer.BYTES - 1, tag, out, out);
+        outputFormat("%08X - %08X  %-30s: 0x%08X         (%d)%n", addr, addr + Integer.BYTES - 1, tag, out, out);
     }
 
     static void output(final long addr, final String tag, final byte out) {
-        System.out.format("%08X - %08X  %-30s: 0x%02X               (%d)%n", addr, addr, tag, out, out);
+        outputFormat("%08X - %08X  %-30s: 0x%02X               (%d)%n", addr, addr, tag, out, out);
     }
 
     static void output(final long addr, final int length, final String tag, final Object out) {
-        System.out.format("%08X - %08X  %-30s: %s%n", addr, addr + length - 1, tag, out.toString());
+        outputFormat("%08X - %08X  %-30s: %s%n", addr, addr + length - 1, tag, out.toString());
+    }
+
+    private static void outputFormat(String format, Object ... args) {
+        System.out.format(format, args);
```

Comment:
I wonder if we could just suppress the warning about System.out.x for this method, as it's actually outputting like a command line tool should?

Reply:
Actually I really feel we should not do the output format thing because IDEs give you guys warnings if your format string is mismatched with your Params and I imagine the indirection breaks that